NOTICE

```diff
@@ -1,100 +1,13 @@
-Crate Data Admin Interface
+Crate.io Sphinx Theme
+Copyright 2013-2018 CRATE Technology GmbH ("Crate")
 
-Copyright 2013-2015 Crate.IO GmbH
-Prinzessinnenstr. 19-20, 10969 Berlin, Germany
-(referred to in this notice as "Crate")
 
+Licensed to CRATE Technology GmbH (referred to in this notice as "Crate")
```

Comment:
Crate.io Inc?

Reply:
i'm working with naomi on this. IP is in the Crate.IO GmbH